Device.md

```diff
@@ -23,7 +23,9 @@ A Harp Device is expected to implement the following `Operation Modes`:
 - *`Speed Mode:`* Allows the implementation of a different and specific communication protocol. On this mode, the Harp Binary Protocol is no longer used. The specific protocol designed must implement the possibility to exit this mode.
 
 The mandatory Operation Modes are the **`Standby Mode`** and **`Active Mode`**. The **`Speed Mode`** is optional and, in many of the applications, not needed.
-It’s strongly recommended that a Harp Device acting as peripheral should continuously check if the communication with the host is active and healthy. If this doesn’t happen over the last 3 seconds, the Harp Device should go to Standby Mode and flush/destroy its TX buffer.
+Harp Devices should continuously check if the communication with the host is active and healthy. This status check will be largely dependent on the transport layer implementing the harp protocol between host and device. Thus, while each implementing must define `Connected` or `Not Connected` status, it is up to the developer to decide how to implement this status check.
```

Comment:
We agreed the definition and implementation of the check should be architecture specific, but do we want to also remove the bit where we say that devices should go to Standby mode when disconnected? These two aspects seem to be orthogonal, i.e. the definition of `Connected` / `Not Connected` states and what happens when a device becomes `Not Connected`.

Reply:
Assuming the above was not intended:

```suggestion
Harp Devices should continuously check if the communication with the host is active and healthy. This status check will be largely dependent on the transport layer implementing the harp protocol between host and device. Each implementation should clearly distinguish between `Connected` and `Not Connected` states, and it is up to the developer to decide how to implement this status check. When the device transitions to the `Not Connected` state, it should immediately enter `Standy Mode` and stop transmission of further event messages.
```

Reply:
Finally, consider whether we should replace the term "should" with "must" and also see #47.

Reply:
Good question. I am just not sure if there is a situation where checking the status of the communication is not possible. For instance, imagine you have some sort of router that is just broadcasting harp messages around, might be difficult to check if there is even anyone listening to those events. At the same time, you could argue that such a broadcaster should not be considered an harp device because it may not be able to receive read/writes since it is just a broadcaster. Not sure...